src/python_testing/TC_IDM_4_2.py

```diff
@@ -287,26 +359,43 @@ async def test_TC_IDM_4_2(self):
                 # Cluster controller 2 has no access to
                 attributes=[(0, Clusters.BasicInformation)],
                 keepSubscriptions=False,
-                reportInterval=[3, 3],
+                reportInterval=(min_interval_floor_sec, max_interval_ceiling_sec),
                 autoResubscribe=False
             )
             raise ValueError("Expected exception not thrown")
         except ChipStackError as e:
```

Comment:
If you check for InteractionModelError here, you should be able to use Status.InvalidAction rather than redefining it, I think.

Reply:
I thought so too, but it only catches ChipStackError 🤷🏼.